docker/test/integration/minifi/core/ImageStore.py

```diff
@@ -181,6 +185,12 @@ def __build_mqtt_broker_image(self):
     def __build_splunk_image(self):
         return self.__build_image_by_path(self.test_dir + "/resources/splunk-hec", 'minifi-splunk')
 
+    def __build_syslog_udp_client_image(self):
+        return self.__build_image_by_path(self.test_dir + "/resources/syslog-client/udp", 'syslog-udp-client')
+
+    def __build_syslog_tcp_client_image(self):
+        return self.__build_image_by_path(self.test_dir + "/resources/syslog-client/tcp", 'syslog-tcp-client')
```

Comment:
Do these need to be built as new images? Looking at the Dockerfiles it seems to me that we could use the ubuntu image and define the entrypoint parameter with the command in the image. This way we could remove the images, and have only 1 container with 2 different commands for TCP and UDP.

Reply:
Good idea. I've changed this in https://github.com/apache/nifi-minifi-cpp/pull/1294/commits/26053fde5f0a6d4ed055ee98558d9653fd89254a

Reply:
Replaced it with a simpler version, based on our conversation :+1:  https://github.com/apache/nifi-minifi-cpp/pull/1294/commits/29d3690e8bd030a1315f6c72fbac0f1a3d0c2cc8